### ISSUE TITLE
Cleanup "Unused" warnings from maven-bundle-plugin

### ISSUE THREAD

**Issue by joakime** [eclipse/jetty.project#2109]
There are hundreds of maven [WARNING] for various unused imports and exports during the build reported by the maven-bundle-plugin.

We should clean them up.

Some Examples
```
[WARNING] Manifest org.eclipse.jetty:jetty-io:jar:9.4.9-SNAPSHOT : Unused Import-Package instructions: [javax.servlet*, javax.transaction*] 
[WARNING] Manifest org.eclipse.jetty.http2:http2-alpn-tests:jar:9.4.9-SNAPSHOT : Unused Export-Package instructions: [org.eclipse.jetty.http2.alpn.tests.*] 
[WARNING] Manifest org.eclipse.jetty.http2:http2-alpn-tests:jar:9.4.9-SNAPSHOT : Unused Import-Package instructions: [javax.servlet*, javax.transaction*, org.eclipse.jetty*] 
[WARNING] Manifest org.eclipse.jetty.fcgi:fcgi-client:jar:9.4.9-SNAPSHOT : Unused Import-Package instructions: [javax.servlet*, javax.transaction*] 
[WARNING] Manifest org.eclipse.jetty.fcgi:fcgi-client:jar:9.4.9-SNAPSHOT : Export org.eclipse.jetty.fcgi.client.http,  has 1,  private references [org.eclipse.jetty.fcgi.generator], 
```

Tip: You can find the various "Unused Import" and "Unused Export" with a `Ctrl+F` of the[ full build log from CI.](https://jenkins.webtide.net/job/jetty.project/job/jetty-9.4.x/1140/consoleFull)

**Comment by janbartel** [comment]
The ones related to the import of javax.servlet and javax.transaction are due to the changes made for this bug:  https://bugs.eclipse.org/bugs/show_bug.cgi?id=477385

It's just saying that those packages aren't used by those modules. I guess we could remove the javax.servlet and javax.transaction import statements.

Not sure what is causing the export warnings. I'll look into it.

**Comment by joakime** [comment]
Heh, that old eclipse bug is going to be reintroduced with the SpinLock being removed again by @sbordet recently.

**Comment by janbartel** [comment]
Fixed for jetty-9.4.10.
